tailwind.config.cjs

```diff
@@ -123,5 +124,5 @@ module.exports = {
       },
     },
   },
-  plugins: [require("@tailwindcss/typography"), require("tailwindcss-animate")],
+  plugins: [require("@tailwindcss/typography"), require("tailwindcss-animate"), require('flowbite/plugin')],
```

Comment:
No need to install another library; you can use Headless UI for the modal.

Reply:
you are using Headless UI 1.7
but the latest version is 2.2
I think flowbite is better than headlessui.
https://npmtrends.com/flowbite-vs-headlessui-vs-tailwind-elements-vs-tailwindcss

Reply:
But we are using Headless UI everywhere, so use that. The version will not affect any required functionality—it's just a modal, nothing major.